cmd/mattermost/commands/command.go

```diff
@@ -67,3 +78,38 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func deleteCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Enter the commandID to delete")
+	}
+
+	confirmFlag, _ := command.Flags().GetBool("confirm")
```

Comment:
I think, for this case, is not necessary the confirm flag, is not a "very destructive" action by itself, and is not "cascading" to destroy any other data. What do you thing about it @wiersgallak ?

Reply:
I would agree.  Also, you have to enter in the Command ID, so it is very specific in its requirements.

Reply:
Done.